integrations/oci/sdk/runtime/src/main/java/io/helidon/integrations/oci/sdk/runtime/OciExtension.java

```diff
@@ -121,32 +125,59 @@ private OciExtension() {
     }
 
     /**
-     * Returns the {@link OciConfig} that is currently defined in the bootstrap environment. If one is not defined under
-     * config {@value OciConfig#CONFIG_KEY} then a default implementation will be constructed.
+     * Returns the global {@link OciConfig} that is currently defined in the bootstrap environment.
+     * <p>
+     * The implementation will first look for an {@code oci.yaml} file, and if found will use that file to establish the global
+     * oci-specific bootstrap {@link io.helidon.config.spi.ConfigSource}.
+     * <p>
+     * If the implementation is unable to find this file, then a fallback mechanism will be used to find it in the configuration
+     * found in the {@link InjectionServices#globalBootstrap()}, using a top-level attribute key named
+     * {@value OciConfigBlueprint#CONFIG_KEY}.
+     * <p>
+     * The final fallback mechanism will use an {@code auto} authentication strategy - see {@link OciConfigBlueprint} for details.
      *
      * @return the bootstrap oci config bean
+     * @see OciConfigBlueprint
      */
     public static OciConfig ociConfig() {
```

Comment:
It seems like any app/api that wants to make calls to OCI will need to call this method first and then use the config to create auth details etc. We should look at doing this only once and not read file/default source to create config object again.

Reply:
pushed.